docs/devops.md

```diff
@@ -4,19 +4,76 @@ title: DevOps
 sidebar_label: DevOps
 ---
 
-At EOS Costa Rica we follow the DevOps principles to build and deploy software. 
+DevOps is the practice & culture of applying lean manufacturing principles to the IT Value stream to increase software velocity, reduce waste, & improve quality with an emphasis on leveraging *people, processes, & tools.*
 
-Automation is at the top of our priorities, everything from scaffolding to code optimization, testing, deploy and delivery.
+It focuses on unifying Development & IT Operations through the principles of flow, feedback, & experimentation.
 
-Continuous integration allows us to detect problems early reducing costs and help us deliver robust software more rapidly.
+<figure class="video_container">
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/OOGFxVzW35w" frameborder="0" allowfullscreen="true">
+  </iframe>
+</figure>
+
+## Key Definitions
+
+- **Continuous Integration (CI)**: Addresses building software from contributions of individual developers in a controlled, agile way. (Checking all code into centralized repo daily)
+- **Continuous Deployment (CD)**: Automated Deployment (or transport) of code to production with no manual steps required. This will not be the plan with all applications.
+- **Continuous Delivery (CD)**: Automation that ensures deployment is validated, however manual step still required to deploy to production.
+- **Deployment**: The deployment (or transport) of code to a higher-level environment, up to & including production.
+- **Release**: The defined scope and business objective for a set of features to be delivered in a certain timeline.
+
+## Why Implement DevOps?
+
+In 2012, Puppet Labs “state of DevOps report” benchmarked 4,039 IT orgs using DevOps and found:
+
+- 30x more frequent code deployments
+- 8,000x faster code deployment lead time
+- 2x the change success rate
+- 12x faster MTTR (mean time to repair)
+
+***
+
+>“*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”*
+
+***
+
+**-Phoenix Project**
```

Comment:
Let's add `**-Phoenix Project**` inside the quote block like this : 

***
>“*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”*
**-Phoenix Project**
***